accountAPI.yaml

```diff
@@ -388,6 +388,186 @@ paths:
+        '501':
+          $ref: '#/components/responses/standard501'
+        '503':
+          $ref: '#/components/responses/standard503'
+
+  /iso20022/{accountId}/notifications:
+    get:
+      tags:
+        - iso20022
+      summary: Retrieve a list of resource links to credit and debit notifications (camt.054)
+      description: >
+        Retrieves the resources links to available credit and debit notifications (camt.054),
+        which conform to Swiss Payment Standards and ISO20022 XML schema guidelines.
+      parameters:
+        - $ref: '#/components/parameters/authorization_in_header'
+        - $ref: '#/components/parameters/path_accountId'
+        - $ref: '#/components/parameters/clientid_in_header'
+        - $ref: '#/components/parameters/correlation_in_header'
+        - $ref: '#/components/parameters/agent_in_header'
+        - $ref: '#/components/parameters/optional_targetid_in_header'
+        - $ref: '#/components/parameters/optional_psu_ip_in_header'
+        - $ref: '#/components/parameters/optional_psu_user_agent_in_header'
+      responses:
+        '200':
+          description: Returns a list of resource links to ISO20022 XML camt.054 messages.
+          headers:
+            X-Correlation-ID:
+              $ref: '#/components/headers/X-Correlation-ID'
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Iso20022ReportReference054'
+        '400':
+          $ref: '#/components/responses/standard400'
+        '401':
+          $ref: '#/components/responses/standard401'
+        '403':
+          $ref: '#/components/responses/standard403'
+        '404':
+          $ref: '#/components/responses/standard404'
+        '405':
+          $ref: '#/components/responses/standard405'
+        '500':
+          $ref: '#/components/responses/standard500'
+        '501':
+          $ref: '#/components/responses/standard501'
+        '503':
+          $ref: '#/components/responses/standard503'
+
+  /iso20022/{accountId}/notifications/{reportId}:
+    get:
+      tags:
+        - iso20022
+      summary: Retrieves the designated ISO20022 XML CAMT document.
+      parameters:
+        - $ref: '#/components/parameters/path_reportId'
+        - $ref: '#/components/parameters/authorization_in_header'
+        - $ref: '#/components/parameters/path_accountId'
+        - $ref: '#/components/parameters/clientid_in_header'
+        - $ref: '#/components/parameters/correlation_in_header'
+        - $ref: '#/components/parameters/agent_in_header'
+        - $ref: '#/components/parameters/optional_targetid_in_header'
+        - $ref: '#/components/parameters/optional_psu_ip_in_header'
+        - $ref: '#/components/parameters/optional_psu_user_agent_in_header'
+      responses:
+        '200':
+          description: Returns the requested ISO20022 XML camt.052 message.
```

Comment:
@simonmeyerm In the description there's a type. camt.052 should be replaced by camt.054.

Reply:
Thank you for the review. Description is now adjusted